jolly-jellyfish/src/static/themes/unicorn.css

```diff
@@ -7,17 +7,18 @@
 @import url('https://fonts.googleapis.com/css2?family=Sacramento&display=swap');
 
 body {
-  background: url("/static/images/unicorn0.gif") left top / 33% 33% no-repeat,
-              url("/static/images/unicorn1.gif") left center / 33% 33% no-repeat,
-              url("/static/images/unicorn2.gif") left bottom / 33% 33% no-repeat,
-              url("/static/images/unicorn3.gif") center top / 33% 33% no-repeat,
-              url("/static/images/unicorn4.gif") center center / 33% 33% no-repeat,
-              url("/static/images/unicorn5.gif") center bottom / 33% 33% no-repeat,
-              url("/static/images/unicorn6.gif") right top / 33% 33% no-repeat,
-              url("/static/images/unicorn7.gif") right center / 33% 33% no-repeat,
-              url("/static/images/unicorn8.gif") right bottom / 33% 33% no-repeat,
-              url("/static/images/unicorn9.gif") left top / 33% 33% no-repeat,
-              linear-gradient(to right, red, orange, yellow, green, blue, indigo, violet) 100% 100% repeat;
+  background:
+    url("../images/unicorn0.gif") left top / 33% 33% no-repeat,
+    url("../images/unicorn1.gif") left center / 33% 33% no-repeat,
+    url("../images/unicorn2.gif") left bottom / 33% 33% no-repeat,
+    url("../images/unicorn3.gif") center top / 33% 33% no-repeat,
+    url("../images/unicorn4.gif") center center / 33% 33% no-repeat,
+    url("../images/unicorn5.gif") center bottom / 33% 33% no-repeat,
+    url("../images/unicorn6.gif") right top / 33% 33% no-repeat,
+    url("../images/unicorn7.gif") right center / 33% 33% no-repeat,
+    url("../images/unicorn8.gif") right bottom / 33% 33% no-repeat,
+    url("../images/unicorn9.gif") left top / 33% 33% no-repeat,
+    linear-gradient(to right, red, orange, yellow, green, blue, indigo, violet) 100% 100% repeat;
```

Comment:
Not sure how much I like the small amount of indentation here... Maybe indent more? (there was another spot with this problem too... but I already passed it xd)

Reply:
I will go back to the visual indent, sure